app/assets/javascripts/directives/views/historyMenu.ts

```diff
@@ -103,16 +122,14 @@ class SessionHistoryMenuCtrl implements SessionHistoryScope {
     };
     if (!this.application.historyManager!.isDiskEnabled()) {
       this.application.alertService!.confirm(
-        `Are you sure you want to save history to disk? This will decrease general 
-        performance, especially as you type. You are advised to disable this feature 
-        if you experience any lagging.`, 
-        undefined,
-        undefined,
-        undefined,
-        run,
-        undefined,
-        true, 
-      );
+        "Are you sure you want to save history to disk? This will decrease general " +
+        "performance, especially as you type. You are advised to disable this feature " +
+        "if you experience any lagging."
```

Comment:
I think we lost the `danger` button accent in the conversion. It's the `true` argument at the end of the method call. BTW, since we're in the web repo you can use the `confirmDialog` function instead which supports named arguments specifically to avoid this kind of confusion. There should be some examples of it in the codebase already.